src/app/[locale]/preferences/components/NotificationSettings.tsx

```diff
@@ -64,50 +61,32 @@ const NotificationSettings = ({ notificationSetting, setNotificationSetting }: P
     }
 
     return (
-        <>
+        <Accordion defaultActiveKey='Project'>
             {Object.values(PREFERENCES).map((value) => (
-                <div className={styles.section} key={value.documentType}>
-                    <div
-                        className={`${styles['section-header']} ${
-                            selectedType === value.documentType && styles.selected
-                        }`}
-                        id={`${value.documentType}Heading`}
-                        onClick={() => {
-                            setSelectedType(selectedType === value.documentType ? 'None' : value.documentType)
-                        }}
-                    >
-                        {value.documentType}
-                    </div>
-                    <div
-                        id={value.documentType}
-                        className={`${styles['body-wrapper']} ${
-                            selectedType === value.documentType ? styles.show : styles.collapse
-                        }`}
-                    >
-                        <div className={`${styles['section-body']}`}>
-                            {Object.values(value.setting).map((entry) => (
-                                <div className='form-group' key={entry.id}>
-                                    <div className='form-check'>
-                                        <input
-                                            id={entry.id}
+                <>
+                    <Accordion.Item eventKey={value.documentType}>
+                        <Accordion.Header>{value.documentType}</Accordion.Header>
+                        <Accordion.Body>
+                            <Form name={value.documentType}>
```

Comment:
Move this Form to div. This causes some errors
![image](https://github.com/eclipse-sw360/sw360-frontend/assets/94659621/a85a6615-a150-49d8-b39b-cd4fc0855763)

Reply:
Fixed, as the entries were never returned before SSR is rendered.
No need for move back to div.

Reply:
Fixed